pkg/utils/util.go

```diff
@@ -91,3 +94,19 @@ func RunCommand(name string, args ...string) {
 	err := cmd.Run()
 	CheckIfError(err)
 }
+
+// Download Downloads a file.
+func Download(url string) ([]byte, error) {
+	resp, err := http.Get(url)
+	if err != nil {
+		return nil, err
+	}
+
+	defer func() { _ = resp.Body.Close() }()
```

Comment:
why is it necessary to put this in a func()

Reply:
Because then there will be linter warning about unhandled error. When we use defer, we should put it in function that we can handle or ignore the error.